### PR TITLE
CORE-2236 updates after moving CF config to http and more

### PR DESCRIPTION
**Pull request by DavidCockerill**

### CODE REVIEW

File: docs/release-notes/4.tucker/4.2.0.md [hunk 1]
```diff
@@ -33,6 +33,25 @@ HarperDB also includes support for Server-Sent Events. This is a very easy-to-us
...
+* Made `operationsApi` configuration optional. Any config not defined here will default to the `http` section.
+* Added a `securePort` parameter used for setting the https port.
+* Added a new top level `tls` section.
+* Broke out the `customFunction` configuration into the `http` and `tls` sections.
+* Removed `customFunctions.enabled`, `customFunctions.network.https`, `operationsApi.network.https` and `operationsApi.nodeEnv`
+* Added a element called `componentRoot` which replaces `customFunctions.root`.
+* Updated custom pathing to use `databases` instead of `schemas`.
+
+## Alpha Status (Current Progress)
+This is alpha release software; it is still in development with active testing. Most of the functionality in this release is working in its primary use cases, but there are many edge cases to test and refinements to finish. Here are a list of the major development works in progress:
```
Comment (kriszyp) [inline docs/release-notes/4.tucker/4.2.0.md:49]: I don't know if you intended to add this section back in? (I think I had removed the alpha section)

**Review by kriszyp — APPROVED** [review]

File: docs/release-notes/4.tucker/4.2.0.md (hunk 1 above)
Comment (DavidCockerill) [reply docs/release-notes/4.tucker/4.2.0.md]: sorry, bad merge. Have removed.
